### DISCUSSION TITLE
Help with Ebus Adapter 3.1 change from USB Mode (factory setting) to Ethernet Mode needed

### DISCUSSION THREAD

**Discussion by karusell** [john30/ebusd#1116] [Q&A]
Hello everyone,

Is it possible to reconfigure a eBus Adapter V3.1 [..] 07/2022 with factory setting "USB MODE" with Jumpers to the "Ethernet MODE"

**Question:** Or is the "Software Jumper" leading and the reconfiguration is only with ebuspicloader possible and no longer with hardware jumpers?

Changelog says "fix: hardware jumpers no longer working since 20221206"
[https://adapter.ebusd.eu/v31/firmware/ChangeLog#version-20221206-reported-as-1-7f16](url)

My eBus Adapter has been delivered i 01/2023 maybe i got already the Firmware that is not supporting longer hardware jumpers?

As @john30  descripted in the documentation for V3.1 it should be possible for this version of plantine to set jumpers and get the **Ethernet Mode**. see picture:
<img width="128" alt="smd31-jumperf" src="https://github.com/john30/ebusd/assets/43848891/e0ed8804-90f0-43e8-985b-eaa69816676b">

According to this I used the following jumpers for J12:  

- **Pin 2 and 3** connected to get **standard protocol**
- **Pin 5 and 6** connected to get **Ethernet MODE**

![my platine](https://github.com/john30/ebusd/assets/43848891/f7657e63-ac21-4e30-a9f8-520f8d22e047)

I bought USR-ES1 W5500 module in Amazon.

Is there something else to do to get the Ethernet mode via Hardware Jumpers for V3.1 07/2022 ?

Hope you could give me some help / ideas. 

PS: to change with ebuspicloader i tried also before but my PC is unable to open /dev/ttyS*.
I have same Problem with this as someone else discussed here: #792 

I tried with two different PCs. 
Question: Is there a USB driver needed which is not mentioned in the documentation?

Thank you everyone.

Regards K.

**Comment by john30** [comment]
depending on the settings done via ebuspicloader it might ignore the jumpers, just check the actual+possible settings with ebuspicloader
